build.xml

```diff
@@ -97,7 +106,9 @@
         <drush command="site-install" assume="yes">
             <option name="account-name">admin</option>
             <option name="account-pass">admin</option>
+            <option name="site-name">${site.name}</option>
             <param>config_installer</param>
+            <param>config_installer_sync_configure_form.sync_directory=../../../config/sites/d8</param>
```

Comment:
This should be set in the site's `settings.php` instead of sneaking it into the config installer:

```php
$settings['config_sync_directory'] = '../../../config/sites/d8';
```

Reply:
I have had the worst luck with all of that...

Reply:
It's there as well. The available documentation on these processes (from Drupal.org) is absolutely crap.